src/assets/icons/action-figure/actionfigure.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { SVGProps } from 'react';
+const ActionFigure = (props: SVGProps<SVGSVGElement>) => (
+	<svg
+		xmlns="http://www.w3.org/2000/svg"
+		fill="none"
+		viewBox="0 0 42 42"
+		{...props}
+	>
+		<path
+			fill="#1E4C2F"
```

Comment:
Icons shouldn't have their own color, they should take whatever color assigned by the consuming component.
To make this possible, we should use `currentColor` instead of hard coded colors. Please check every icon and make sure they have `currentColor` value set instead of hard coded colors.

Reply:
Noted 
We'd also make changes to that